### PR TITLE
Fix dashboard device boxes

### PR DESCRIPTION
**Pull request by atrovato**
Showcase on PR #1029 allows to find 2 bugs on dashboard device boxes:
1. dimmer doesn't update the right device index
2. color 0 (black) introduce a color picker bug

### CODE REVIEW

**Review by Pierre-Gilles — APPROVED** [review]
Nice catch!

File: front/src/components/boxs/device-in-room/device-features/ColorDeviceFeature.jsx [hunk 1]
```diff
@@ -64,7 +64,7 @@ class ColorDeviceType extends Component {
 
   componentDidMount() {
     const deviceLastValue = this.props.deviceFeature.last_value;
-    const color = deviceLastValue === null ? undefined : `#${intToHex(deviceLastValue)}`;
+    const color = !deviceLastValue ? undefined : `#${intToHex(deviceLastValue)}`;
```
Comment (Pierre-Gilles) [inline front/src/components/boxs/device-in-room/device-features/ColorDeviceFeature.jsx:67]: Mmm wait, does it means that when the color is 0, it shows an `undefined`color ? Shouldn't it be the black hexadecimal color?

Comment (atrovato) [reply front/src/components/boxs/device-in-room/device-features/ColorDeviceFeature.jsx:67]: The color pocker doesn't have black color, it causes trouble, and I don't think your light has any black color too.
Black means switched off, it is not really a color.

Comment (Pierre-Gilles) [reply front/src/components/boxs/device-in-room/device-features/ColorDeviceFeature.jsx:67]: True!

**Pull request merged**